gear-programs/bridging-payment-vara-supply/src/services/utils.rs

```diff
@@ -0,0 +1,97 @@
+use super::{
+    error::Error,
+    msg_tracker::{msg_tracker_mut, MessageStatus},
+    vft_treasury::vft_treasury::io as vft_treasury_io,
+    vft_treasury::Error as VftTreasuryError,
+};
+use sails_rs::calls::ActionIo;
+use sails_rs::prelude::*;
+
+macro_rules! maybe_event_or_panic_async {
```

Comment:
can you re-write this as fn or remove it? It's used only in 2 places and it's hard to understand this logic outside of the context of `request_tx` or `continue_tx`

Reply:
Fixed